doc/api/util.md

```diff
@@ -398,6 +398,83 @@ util.inspect.defaultOptions.maxArrayLength = null;
 console.log(arr); // logs the full array
 ```
 
+## util.promisify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function}
+
+Takes a function following the common Node.js callback style, i.e. taking a
+`(err, value) => ...` callback as the last argument, and returns a version
+that returns promises.
```

Comment:
What happens if a Sync function is passed to this? Is the behavior undefined? Is there any way that we could define any reasonable behavior? (I doubt it... I'm +1 on simply saying that the behavior is undefined)